lib/samples/display_clusters/display_clusters_sample.dart

```diff
@@ -25,28 +25,31 @@ class DisplayClustersSample extends StatefulWidget {
 }
 
 class _DisplayClustersSampleState extends State<DisplayClustersSample> {
+  // create a map view controller
   final _mapViewController = ArcGISMapView.createController();
-  final _map = ArcGISMap.withUri(
-    Uri.parse(
-        'https://www.arcgis.com/home/item.html?id=8916d50c44c746c1aafae001552bad23'),
-  )!;
-
+  // creates a map
+  late ArcGISMap _map;
+  // create a flag to check if the map is ready
   bool _ready = false;
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       body: SafeArea(
+        top: false,
+        // create a column widget
         child: Column(
           children: [
             Expanded(
+              // add a map view to the widget tree and set a controller.
               child: ArcGISMapView(
                 controllerProvider: () => _mapViewController,
                 onMapViewReady: onMapViewReady,
               ),
             ),
             Center(
-              child: TextButton(
+              // create a button to toggle feature clustering
```

Comment:
I realized we are using ElevatedButton everywhere and updated this sample accordingly. Please check the new screenshot for the new button. Is it fine?
@PaulAllanSturm

Reply:
Yes, looks good.

I added a note to the standards list that we should always use ElevatedButton.